tests/input/elements.xml

```diff
@@ -181,7 +181,8 @@
             <li>
                Text and format="default": Sections
                <xref target="RFC7997" format="default" sectionFormat="bare" section="3.2">Person Names</xref> and 
-               <xref target="RFC7997" format="default" sectionFormat="bare" section="3.3">Company Names</xref> of
+               <xref target="RFC7997" format="default" sectionFormat="bare" section="3.3">Company Names</xref> and
+               <xref target="RFC7997" format="default" sectionFormat="bare" section="3.3">a <em>lot</em> <strong>other</strong> information</xref> of
```

Comment:
nit - it would be less eye-catching if this said "a lot more" or "a lot of other" rather than "a lot other".

Reply:
ah. omission is a typo.

Reply:
Now the test says `<xref target="RFC7997" format="default" sectionFormat="bare" section="3.3">a <em>lot</em> <strong>more</strong> information</xref> of`